frontend/src/app/organization/organization.service.ts

```diff
@@ -14,10 +16,12 @@ export class OrganizationService {
   private contactStatusEvent = new BehaviorSubject<ContactStatus | null>(null);
   private organizationEvent = new BehaviorSubject<Organization | null>(null);
   private organizationsEvent = new BehaviorSubject<Organization[] | null>(null);
+  private offersEvent = new Subject<Offer[]>();
 
   public contactStatus$: Observable<ContactStatus | null> = this.contactStatusEvent.asObservable();
   public organization$: Observable<Organization | null> = this.organizationEvent.asObservable();
   public organizations$: Observable<Organization[] | null> = this.organizationsEvent.asObservable();
+  public offers$: Observable<Offer[] | null> = this.offersEvent.asObservable();
```

Comment:
It is possible to get empty array but not `null` value. So I should use `Subject` and I shouldn't initialize its value with `null`?

Reply:
Let's talk about the difference between BehaviourSubject and Subject. BehaviourSubject always keeps the last value that was passed to it. When you subscribe to BehaviourSubject, you instantly get the last value, then the suscription waits for events. With Subject, you just subscribe and wait for events, it doesn't care about what happend in the time before subscription. That's why when constructing BehaviourSubject you need to pass the initial value to constructor (doesn't have to be null, can be some object with values), so that you can retrieve the value when you subscribe for the first time. In your current solution we don't need to hold the last value, we just pass the result to the component, so Subject is enough and it will not be nullable since there is always array returned from the backend.